### PR TITLE
SilverStripe 4 support

### PR DESCRIPTION
**Pull request by ntd**
A serie of commits that integrates and improves #18.

### CODE REVIEW

**Comment by ntd** [comment]
To complete the port [PSR-2](https://www.php-fig.org/psr/psr-2/) coding style should be applied. I'd like to perform also some minor clean up, such as removing the execution bit from the files, but I'd prefer to have a somewhat working version before proceeding.

File: code/Silvergraph.php [hunk 1]
```diff
@@ -51,7 +60,7 @@ public function dot(){
         $renderClasses = array();
 
         //Get all DataObject subclasses
-        $dataClasses = ClassInfo::subclassesFor('DataObject');
+        $dataClasses = ClassInfo::subclassesFor('SilverStripe\\ORM\\DataObject');
```
Comment (Firesphere) [inline code/Silvergraph.php]: You could do this with `DataObject::class`

File: code/Silvergraph.php [hunk 2]
```diff
@@ -204,7 +200,7 @@ public function dot(){
 
         // Defend against source_file_comments
         Config::nest();
-        Config::inst()->update('SSViewer', 'source_file_comments', false);
+        Config::inst()->update('SilverStripe\\View\\SSViewer', 'source_file_comments', false);
```
Comment (Firesphere) [inline code/Silvergraph.php]: As above

**Comment by Firesphere** [comment]
I would suggest changing the `define()` to use `Environment::get()`, so the paths can be set in .env files

File: code/Silvergraph.php (hunk 2 above)
Comment (ntd) [reply code/Silvergraph.php]: This is not exactly the same (there is no `use SilverStripe\View\SSViewer` before) but I think that making that dependency explicit is a good move anyway. :+1:

File: code/Silvergraph.php (hunk 1 above)
Comment (ntd) [reply code/Silvergraph.php]: :+1:

**Pull request merged**